lib/phoenix/digester.ex

```diff
@@ -46,18 +47,48 @@ defmodule Phoenix.Digester do
     |> Enum.map(&(map_file(&1, input_path)))
   end
 
-  defp generate_manifest(files, output_path) do
+  defp load_digests(input_path) do
+    manifest_path = Path.join(input_path, "manifest.json")
```

Comment:
I believe the `manifest.json` is in the output path, where this ultimately writes it to, which means that this doesn't need the `input_path` which means it can probably be called only by `generate_manifest` (which can then take its original two arguments only.